### DISCUSSION TITLE
채팅방 입장하기

### DISCUSSION THREAD

**Discussion by disabled-variable** [storycraft/node-kakao#686] [Q&A]
![image](https://user-images.githubusercontent.com/84914862/134907256-a5c1c439-804a-4304-80b2-1456aec5d747.png)
채널ID를 기반으로 특정 채팅방에 입장하고자 다음과 같이 구성했습니다.
첫줄만 실행시에는 오류가 나지않는데, 함께 실행하면 오류가 발생합니다.

![image](https://user-images.githubusercontent.com/84914862/134907330-4adf9bb2-254b-41a5-8760-04cda171cbae.png)
오류 메세지는 다음과 같습니다.

----------------------------------------------------------------------------------------------------------------------------------------------

특정 채팅방(일대일 개인채팅, 오픈채팅)에 입장하는 함수와 매게변수 값이 궁금합니다.

**Comment by generalxd** [comment]
channelList.open.joinChannel

Typescript 시면 컨트롤 스페이스 누르면 요구 인자 목록 출력됩니다.
